nightly-job/query_sanitization.py

```diff
@@ -163,6 +165,33 @@ def stream_search_terms():
     # df_generator = query_job.result(page_size=75000).to_dataframe_iterable()
     return df_generator
 
+
+UNSANITIZED_QUERY_STATS = """
+SELECT
+    COUNT(DISTINCT search_logs.jsonPayload.fields.rid) AS total_term_count,
+    COUNTIF(TRIM(search_logs.jsonPayload.fields.query) = "") AS total_blank_count,
+FROM `suggest-searches-prod-a30f.logs.stdout` AS search_logs
+WHERE 
+    jsonPayload.type = "web.suggest.request"
+    --Specifically get the previous day's data
+    AND timestamp >= DATE_ADD(CURRENT_TIMESTAMP(), INTERVAL -1 DAY)
+    AND DATE(timestamp) < CURRENT_DATE()
```

Comment:
We should change the date filter to `DATE(timestamp) = DATE_ADD(CURRENT_DATE(), INTERVAL -1 DAY)` here and above.

This is a separate issue I noticed, where the current date filter is dependent on the job start time. I mentioned this in [DENG-3279](https://mozilla-hub.atlassian.net/browse/DENG-3279) but I should have opened a separate ticket.

Reply:
It _is_, and if I recall correctly we did it like this for a specific reason, but for the life of me now I can't recall the reason 😅

Reply:
Ok yeah, if this is necessary we should discuss or document - it does come at the cost of silently dropping all search terms submitted betweeen midnight and the job start time.

Reply:
Updated the date logic in my most recent commit.